python/lsst/pipe/tasks/measurementDriver.py

```diff
@@ -1206,8 +1266,19 @@ def _deblendSources(
         """
         self.log.info(f"Deblending using '{self._Deblender}' on {len(catalog)} detection footprints")
 
+        if mDeconvolved is None:
+            # Make a deconvolve version of the multi-band exposure.
+            deconvolvedCoadds = []
+            deconvolveTask = DeconvolveExposureTask()
```

Comment:
Is/should there be any way to configure the task?

Reply:
I believe so, but it's something for the future. We might even want to make it a subtask.